lib/temporal/connection/converter/codec/chain.rb

```diff
@@ -0,0 +1,36 @@
+require 'temporal/connection/converter/codec/base'
+
+module Temporal
+  module Connection
+    module Converter
+      module Codec
+        # Performs encoding/decoding on the payloads via the given payload codecs. When encoding
+        # the codecs are applied last to first meaning the earlier encodings wrap the later ones.
+        # When decoding, the codecs are applied first to last to reverse the effect.
+        class Chain < Base
+          def initialize(payload_codecs:)
```

Comment:
nit: This might be more ergonomic with a splat or non-keyword argument instead

Reply:
i agree, i was just keeping the same pattern as https://github.com/coinbase/temporal-ruby/blob/378630a545ac52759ac1030397f66aebfdec24df/lib/temporal/connection/converter/composite.rb#L10, what do you think?